front/package-lock.json

```diff
@@ -56,6 +57,31 @@
         "typescript": "^5.0.3"
       }
     },
+    "../connectors": {
+      "version": "0.1.2",
```

Comment:
When running `npm install`  from `front`:
```
spolu@spolu-dev ~/code/dust/front (aric/connectors-sub-project) $ git diff
diff --git a/front/package-lock.json b/front/package-lock.json
index 6252284..d45cab2 100644
--- a/front/package-lock.json
+++ b/front/package-lock.json
@@ -58,6 +58,7 @@
       }
     },
     "../connectors": {
+      "name": "dust-connectors",
       "version": "0.1.2",
       "dependencies": {
         "@slack/web-api": "^6.8.1",
spolu@spolu-dev ~/code/dust/front (aric/connectors-sub-project) $ 
```

Reply:
I guess it is due to the different node versions

Reply:
Solved 👍